src/components/value-list/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { getItemIndex } from "../pick-list/shared-list-logic";
+import { DragStatus } from "./interfaces";
+import { ValueList } from "./value-list";
+
+export function getScreenReaderText(
+  item: HTMLCalciteValueListItemElement,
+  status: DragStatus,
+  valueList: ValueList<HTMLCalciteValueListItemElement>
+): string {
+  const {
+    items,
+    intlDragHandleStart,
+    intlDragHandleActivated,
+    intlDragHandleNewPosition,
+    intlDragHandleCurrentPosition
+  } = valueList;
+  const total = items.length;
+  const position = getItemIndex(valueList, item) + 1;
+  switch (status) {
+    case "start":
+      return intlDragHandleStart
+        ? intlDragHandleStart
+        : `${item.label} ,press space and use arrow keys to re-order content. current position ${position} of ${total}`;
+    case "activated":
+      return intlDragHandleActivated
+        ? intlDragHandleActivated
+        : `Reordering ${item.label} ,current position ${position} of ${total}`;
+    case "newPosition":
+      return intlDragHandleNewPosition
+        ? intlDragHandleNewPosition
+        : `${item.label} , new position ${position} of ${total}. press space to confirm`;
+    case "currentPosition":
+      return intlDragHandleCurrentPosition
+        ? intlDragHandleCurrentPosition
+        : `${item.label} ,current position ${position} of ${total}`;
+    default:
+      break;
+  }
+}
+
+export function getHandleAndItemElement(event: KeyboardEvent | FocusEvent): {
+  handleElement: HTMLCalciteHandleElement;
```

Comment:
Nitpick: the interface includes 2 elements, but the naming is inconsistent. My vote is for `{ handle, item }`.